### PR TITLE
command.h include was not set correctly

### PR DESCRIPTION
**Pull request by drashna**

### CODE REVIEW

File: tmk_core/common/command.h [hunk 1]
```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef COMMAND_H
-#define COMMAND
+#pragma once
```
Comment (vomindoraan) [inline tmk_core/common/command.h]: I suggest changing the offending line to `#define COMMAND_H` instead. The rest of `tmk_core` code uses traditional header guards, and it'd be nice to remain consistent.

I'm all for `#pragma once`, so maybe we can make a separate PR that changes all existing traditional header guards to `#pragma once`? I just wouldn't do it here.

Comment (vomindoraan) [reply tmk_core/common/command.h]: Actually, I'm fine with `#pragma once`. You're right, we don't have to switch everything at once. Baby steps :smile:

**Comment by jackhumbert** [comment]
Thanks!
